# Generated by Django 5.1.5 on 2025-04-14 08:24

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('backend', '0035_failedloginattempt'),
    ]

    operations = [
        migrations.AlterField(
            model_name='workout',
            name='name',
            field=models.CharField(max_length=255, validators=[django.contrib.auth.validators.UnicodeUsernameValidator]),
        ),
    ]
